docker/docker-compose.yaml

```diff
@@ -37,5 +37,13 @@ services:
     ports:
       - 8002:80
 
+  ssh:
+    container_name: 'kernelci-ssh'
+    build:
+      context: ssh
+    volumes:
+      - './storage/data:/home/kernelci/data'
+      - './ssh:/home/kernelci/.ssh'
```

Comment:
```suggestion
      - './ssh/user-data:/home/kernelci/.ssh'
```

Reply:
Oh! I forgot to commit that change.
Just did.